### DISCUSSION TITLE
Built in ATC, benefit?

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#6221] [General]
Hi developer Team,

for me it is not really clear what the benefit of the built in ATC is. I followed the description on your page but I wasn´t able to tune Clearence at EDDS airport. So I didn´t get the clearence on ground. After takeoff it was possible to go for the clearence.

Second  observation, not every waypoints will be connected. Might be dangerous in areas with high hills :-p ..

![grafik](https://user-images.githubusercontent.com/76531893/142912763-9d6bbb4e-f34f-4785-9bea-64d3b96d0f20.png)

What is the reason for that?

greets Klaus

**Comment by MStallen** [comment]
To get clearance, no discontinuties should be in your flightplan. If you've one just delete it.

I had also a skipped waypoint in my flightplan (LFKJ-LONSU-NERAS-ILSR04) same as you. The A/C couldn't follow at that point and flew out of control.

**Reply by Benjozork** [reply]
> To get clearance, no discontinuties should be in your flightplan. If you've one just delete it.

This is untrue. In-game (or IRL for that matter) ATC has no knowledge of discontinuities.

**Reply by MStallen** [reply]
If you plan a flight on SimBrief for instance LFKJ Lonsu LFMN and you request that in the MCDU then you will see a discontinuty in your flightplan. If you go to ATC-ground  you won't get the option to request IFR clearance untill you delete the discontinuty.

**Comment by Benjozork** [comment]
Seems you deleted a MANUAL leg or something? The waypoint is not skipped, you're just asking for a tight turn which makes the FMS turn way in advance of the waypoint.

**Reply by MStallen** [reply]
No, I get this also when I a'm selecting an ILS 04R approach in LFMN. And what is even worse,  as soon as the aircraft reaches that point the AP is losing it and goes out of control
